app/main/appium-method-handler.js

```diff
@@ -214,41 +214,40 @@ export default class AppiumMethodHandler {
       } catch (ign) { }
     }
   }
+}
 
-  static createSession (driver, sender, winId) {
-    const {appiumHandlers} = AppiumMethodHandler;
-    log.info(`Creating method handler for session with window id: ${winId}`);
-    const handler = AppiumMethodHandler.appiumHandlers[winId] = new AppiumMethodHandler(driver, sender);
-    log.info(`The following session window ID's have active sessions: '${JSON.stringify(_.keys(appiumHandlers))}'`);
-    return handler;
-  }
-
-  static killSession (winId, killedByUser) {
-    const {appiumHandlers} = AppiumMethodHandler;
-    const handler = appiumHandlers[winId];
-    delete AppiumMethodHandler.appiumHandlers[winId];
-    log.info(`Killing session for window with id: ${winId}`);
+export function createSession (driver, sender, winId) {
```

Comment:
Had to change from `static` to `export` because Babel 7 doesn't accept the format of exporting objects and then doing destructured imports. You have to explicitly export each named variable.

Reply:
👍